api/src/main/resources/workbench-api.yaml

```diff
@@ -5345,6 +5385,13 @@ definitions:
         format: int64
         description: 'Timestamp when the user first changed from Unregistered to Registered
           data access level. May be null if the user has never done so.'
+      ras_identity_type:
```

Comment:
does this better to be an enum or just string?

Reply:
I would use enum